hackathon/spacecraft/app.config.js

```diff
@@ -40,4 +40,24 @@ module.exports = {
       projectId: "012accc3-4ce5-4bae-9f4d-2f842489f07a",
     },
   },
+  plugins: [
+    [
+      "app-icon-badge",
+      {
+        enabled: true, // enable/ disable the plugin based on the environment (usually disabled for production builds)
```

Comment:
disable for production builds 
https://docs.expo.dev/build-reference/variants/#configuring-development-and-production-variants